instrumentation/grpc/src/it/grpc12/pom.xml

```diff
@@ -47,6 +47,11 @@
       <artifactId>junit</artifactId>
       <version>@junit.version@</version>
     </dependency>
+    <dependency>
+      <groupId>org.mockito</groupId>
+      <artifactId>mockito-core</artifactId>
+      <version>@mockito.version@</version>
```

Comment:
Don't know if I've seen `@` before and not sure how to search for it. Quick pointer to what it is?

Reply:
maven-invoker-plugin isolates the build environment, so you have to opt-in to inherit anything. This syntax means the property is lifted from the invoking environment

http://maven.apache.org/plugins/maven-invoker-plugin/examples/filtering.html